simpleclient_jetty_jdk8/src/main/java/io/prometheus/client/jetty/QueuedThreadPoolStatisticsCollector.java

```diff
@@ -37,7 +37,9 @@ public QueuedThreadPoolStatisticsCollector add(QueuedThreadPool queuedThreadPool
         buildGauge("jetty_queued_thread_pool_threads_idle", "Number of idle threads",
             QueuedThreadPool::getIdleThreads),
         buildGauge("jetty_queued_thread_pool_jobs", "Number of total jobs",
-            QueuedThreadPool::getQueueSize));
+            QueuedThreadPool::getQueueSize),
+        buildGauge("jetty_queued_thread_pool_max", "Max size of thread pool",
```

Comment:
threads_max would seem more in line with the above metrics

Reply:
Perhaps. Or would size be more appropriate?

Reply:
I'd say that it's implicit that it's the maximum size of the threadpool, there's nothing else it can really be the max of.